scalableminds/webknossos
# Don't let toast elapse in background in some browsers

### URL of deployed dev instance (used for testing):
- https://timeouttabnotinbackground.webknossos.xyz/

### Steps to test:
- Use Chrome or sth. Chromium based
- Do anything that takes a bit of time and where toasts are shown in the process, such as:
      - upload a larger dataset and switch the tab
      - wait until the upload is done for a while
      - go back to the WK tab and make sure both toasts are shown at least for a very short time (there should be at least two toasts after switching back)
- create scenarios where error toasts and warning toasts are shown. Make sure that their content is printed to the console, except if the toast content is a more complex react node (containing a link, for example).

### Issues:
- fixes #4719

------
(Please delete unneeded items, merge only when none are left open)
